docs/AUTHENTICATORS.md

```diff
@@ -93,6 +127,26 @@ module Authentication
         # initialization code based on ENV config
       end
 
+      # OPTIONAL
```

Comment:
The `valid?` method had this commented info inside the method definition. Might be nice to match that (or change `valid?` to have the comments above the methods definition) for consistency.